x-pack/legacy/plugins/lens/public/indexpattern_plugin/indexpattern_suggestions.ts

```diff
@@ -364,6 +366,9 @@ export function getDatasourceSuggestionsFromCurrentState(
           columnId =>
             layer.columns[columnId].isBucketed && layer.columns[columnId].dataType === 'date'
         );
+        const timeField = indexPattern.fields.find(
+          ({ name }) => name === indexPattern.timeFieldName
+        );
```

Comment:
This change seems good but unrelated, right?

Reply:
It's related in that I needed to compute this in several places, and it made more sense to get it once and pass it down.